<launch>

	<!-- Topic Names -->
	<param name="box_plan_service"						type="string"	value="box_plan" />
	<param name="map_topic"								type="string"	value="/map" />
	<param name="grid_topic"							type="string"	value="/grid" />
	<param name="grid_marker_topic"						type="string"	value="/grid_marker" />
	<param name="grid_markerini_robot_markers_topic"	type="string"	value="/ini_robot_markers" />
	<param name="grid_markerini_boxes_markers_topic"	type="string"	value="/ini_boxes_markers" />
	<param name="grid_markerend_boxes_markers_topic"	type="string"	value="/end_boxes_markers" />
	<param name="grid_markercur_boxes_markers_topic"	type="string"	value="/cur_boxes_markers" />

	<!-- Planner Node -->
	<node pkg="box" name="box_plan" type="box_plan" respawn="false" output="screen">
		<param name="max_iterations"	type="int"		value="1000000000" />
		<param name="time_lim_secs"		type="double"	value="1e3" />
		<param name="epsilon"			type="double"	value="1.0" />
		<param name="verbose"			type="bool"		value="false" />
	</node>

	<!-- Grid Mapper -->
	<node pkg="box" name="grid_mapper" type="grid_mapper.py" respawn="false" output="screen">
		<param name="box_size"			type="double"	value="0.4572" />
	</node>

	<!-- Plan Execution -->
	<node pkg="box" name="box_execute" type="box_execute.py" respawn="false" output="screen">
		<param name="move_base_topic"			type="string"		value="move_base" />
	</node>

	<!-- Rviz -->
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find box)/rviz/box.rviz"/>

</launch>
